src/common/python/utility.py

```diff
@@ -40,18 +40,31 @@ def get_identifier_from_error(event):
 
 def get_modified_identifier(identifier):
```

Comment:
since a new method `get_modified_response` has been added, can this one be removed?